testing/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+
+set -e
+
+echo "Expect to run from docker, in callek/shipit-test-runner"
+echo "  docker run --rm -v `pwd`:/src -ti callek/shipit-test-runner /src/testing/run_tests.sh"
```

Comment:
I don't feel we should make `docker build` a requirement of tox.ini, since tox is inherently more for python than other testing. And some people may wish to use npm/etc locally outside of docker.

I'm not sure the best way though, despite those concerns...